G-Research.OpenTelemetry.Processor.Partial/Example.cs

```diff
@@ -22,19 +24,19 @@ public static void Main()
         {
             Protocol = OtlpExportProtocol.Grpc,
             Endpoint =
-                new Uri("http://otel-partial-collector:4317")
+                new Uri("http://localhost:4317")
         });
 
         var otlpLogExporter = new OtlpLogExporter(new OtlpExporterOptions
         {
             Protocol = OtlpExportProtocol.HttpProtobuf,
-            Endpoint = new Uri("http://otel-partial-collector:4318/v1/logs")
+            Endpoint = new Uri("http://localhost:4318/v1/logs")
         });
 
         var tracerProvider = Sdk.CreateTracerProviderBuilder()
             .AddSource("activitySource")
             .AddProcessor(new PartialActivityProcessor(otlpLogExporter,
-                scheduledDelayMilliseconds: 1000))
+                heartbeatIntervalMs: 1000))
```

Comment:
I would name the property `heartbeatIntervalMilliseconds` as `Milliseconds` seems to be the suffix used for other similar properties in the upstream libs